protocol/src/main/protos/api/api.proto

```diff
@@ -1189,10 +1198,20 @@ message BlockExtention {
   bytes blockid = 3;
 }
 
+message BlockInfo {
```

Comment:
These changes of the protocol need to be synchronized here：  https://github.com/tronprotocol/protocol

Reply:
I'm not sure how to do it :/

Reply:
> I'm not sure how to do it :/
Modify the proto file in the corresponding position of https://github.com/tronprotocol/protocol and submit a PR in the protocol repo.

Reply:
I got this my bad